ingestion/functions/parsing/brazil_goias/goias.py

```diff
@@ -0,0 +1,189 @@
+import os
+import sys
+from datetime import datetime
+import csv
+
+# Layer code, like parsing_lib, is added to the path by AWS.
+# To test locally (e.g. via pytest), we have to modify sys.path.
+# pylint: disable=import-error
+try:
+    import parsing_lib
+except ImportError:
+    sys.path.append(
+        os.path.join(
+            os.path.dirname(os.path.dirname(os.path.abspath(__file__))),
+            "common/python"))
+    import parsing_lib
+
+_AGE = "faixa_etaria"
+_GENDER = "sexo"
+_MUNICIPALITY = "municipio"
+_DATE_CONFIRMED = "data_notificacao"
+_DATE_SYMPTOMS = "data_inicio_sintomas"
+_RECOVERED = "recuperado"
+_ETHNICITY = "raca_cor"
+_DIABETES = "diabetes"
+_LUNG = "doenca_respiratoria"
+_IMMUNOSUPPRESSED = "imunossupressao"
+_CARDIOVASCULAR = "doenca_cardiovascular"
+
+_COMORBIDITIES_MAP = {
+    "doenca_respiratoria": "lung disease",
+    "doenca_cardiovascular": "cardiovascular system disease",
+    "diabetes": "diabetes mellitus"
+}
```

Comment:
I think a lot of these lookups, and the functions that use them, are the same in other Brazil parsers (and maybe Portugal if/when we have one). It'd be good to move them to a common file in `ingestion/functions/parsing/common` so that there's a single place to go to read and update them.

Reply:
Do you mean the comorbidities or the column headers? Or both?

Reply:
Everything, really; there's a "words used in Portuguese health reports" library waiting to be extracted here :).

Reply:
Sure. Shall we merge this parser and then I can go off and do that as it will involve me changing all the other parsers anyway? If you're happy with the rest of it that is.